src/explore-education-statistics-admin/src/services/userService.ts

```diff
@@ -47,6 +47,14 @@ export interface UserInvite {
   userPublicationRoles: { publicationId: string; publicationRole: string }[];
 }
 
+export interface PendingInvite {
+  email: string;
+  name: string;
+  role: string;
```

Comment:
Just wondering if we can narrow the type of `role` here with an enum if we know what it's values might be?